tutorials/spanner-oc/java/src/main/java/com/example/spanner/App.java

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.spanner;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.spanner.Database;
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Key;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerException;
+import com.google.cloud.spanner.SpannerExceptionFactory;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.spanner.admin.database.v1.CreateDatabaseMetadata;
+import io.opencensus.common.Scope;
+import io.opencensus.contrib.grpc.metrics.RpcViews;
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import io.opencensus.exporter.trace.stackdriver.StackdriverTraceConfiguration;
+import io.opencensus.exporter.trace.stackdriver.StackdriverTraceExporter;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.config.TraceConfig;
+import io.opencensus.trace.samplers.Samplers;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * This sample demonstrates how to enable opencensus tracing and stats in cloud spanner client.
+ */
+public class App {
+
+  // [START configChanges]
+  private static final String PROJECT_ID = ServiceOptions.getDefaultProjectId();
+  private static final String INSTANCE_ID = System.getenv( "INSTANCE_ID");
+  private static final String DATABASE_ID = System.getenv( "DATABASE_ID");
+  // [END configChanges]
+
+  // [START config_oc_stackdriver_export]
+  private static void configureOpenCensusExporters() throws IOException {
+    TraceConfig traceConfig = Tracing.getTraceConfig();
+
+    // default sampler is set to Samplers.alwaysSample() for demonstration. In production
+    // or in high QPS environment please use default sampler.
```

Comment:
nit: s/default sampler is set/Sampler is set/  Otherwise it is confusing with default sampler in the second line.

Reply:
Sure